### PR TITLE
Created a modern README.md page for Kong Docs repo!

### PR DESCRIPTION
**Pull request by ssiddhantsharma**
**NOTE**: Please read the CONTRIBUTING.md guidelines before submitting your patch,
and ensure you followed them all:

https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md

### Summary

Refined the readme.md with modern standards (badges and links) and fixed some bugs

### Full changelog

* [Edited some badges]
* [Added a modern contributing tab]

### Issues resolved

Fix #XXX

### Checklist:

- [x] [Commit message & atomicity](https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md#commit-atomicity) checked
- [x] Documentation 
- [x] Spellchecked my updates
- [x] Ready to be merged

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -74,4 +77,11 @@ we refer to collectively as "extensions") in the
 [Kong Hub](https://docs.konghq.com/hub), with documentation hosted
 on the Kong website for ready access.
 
-See [CONTRIBUTING](https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md#contributing-to-kong-documentation-and-the-kong-hub) for more information.
```
Comment (coopr) [inline README.md]: Please include this specific link.

File: README.md [hunk 2]
```diff
@@ -74,4 +77,11 @@ we refer to collectively as "extensions") in the
...
+## Contributing 
+We :heart: contributions from everyone! We welcome [bug reports](https://github.com/Kong/docs.konghq.com/issues/), [bug fixes](https://github.com/Kong/docs.konghq.com/pulls) and feedback on the library is always appreciated. Look at the [Contributor Guidelines](CONTRIBUTING.md) for more information.
```
Comment (coopr) [inline README.md]: Please clarify that while github.com/kong/docs.konghq.com is the right place for bug reports and contributions (not just bug fixes) for the Kong docs site, our other repo https://github.com/Kong/kong is where folks should go for the Kong software itself - linking directly to https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#where-to-seek-for-help is a good idea.

Comment (coopr) [inline README.md]: "feedback on the library" - well, there isn't a "library" per se. Also, feedback should be directed to https://discuss.konghq.com/

**Comment by ssiddhantsharma** [comment]
@coopr Resolved the issues you asked about!

File: README.md [hunk 3]
```diff
@@ -74,4 +77,12 @@ we refer to collectively as "extensions") in the
 [Kong Hub](https://docs.konghq.com/hub), with documentation hosted
 on the Kong website for ready access.
 
-See [CONTRIBUTING](https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md#contributing-to-kong-documentation-and-the-kong-hub) for more information.
```
Comment (coopr) [inline README.md]: This needs to not be deleted - it is specifically about contributing to the Kong Hub.

File: README.md [hunk 4]
```diff
@@ -1,5 +1,8 @@
 # KONG Website
 
+[![Build Status](https://api.travis-ci.org/Kong/docs.konghq.com.svg?branch=master)](https://travis-ci.org/Kong/docs.konghq.com/)
```
Comment (coopr) [inline README.md]: This is showing a build error - but... there is no error?

File: README.md [hunk 5]
```diff
@@ -74,4 +77,12 @@ we refer to collectively as "extensions") in the
...
 on the Kong website for ready access.
 
-See [CONTRIBUTING](https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md#contributing-to-kong-documentation-and-the-kong-hub) for more information.
+## Contributing 
+We :heart: contributions from everyone! We welcome bug reports, bug fixes and [Feedback](https://discuss.konghq.com/) is always welcome and appreciated.
+Look at the [Contributor Guidelines](https://github.com/Kong/docs.konghq.com/blob/master/CONTRIBUTING.md#contributing-to-kong-documentation-and-the-kong-hub) or visit [where-to-seek-for-help](https://github.com/Kong/kong/blob/master/CONTRIBUTING.md#where-to-seek-for-help) for more detailed information about contributing to Kong.
+
+## [![contributions welcome](https://img.shields.io/badge/contributions-welcome-brightgreen.svg?style=flat)](https://github.com/Kong/docs.konghq.com/issues) [![GitHub contributors](https://img.shields.io/github/contributors/Kong/docs.konghq.com.svg)](https://GitHub.com/Kong/docs.konghq.com/graphs/contributors/)
+
+## License
```
Comment (coopr) [inline README.md]: We don't need this section, thanks.

File: README.md (hunk 4 above)
Comment (ssiddhantsharma) [reply README.md]: It is not showing any error, the build is passing successfully!

File: README.md (hunk 3 above)
Comment (ssiddhantsharma) [reply README.md]: Resolved!

File: README.md (hunk 5 above)
Comment (coopr) [reply README.md]: This matter is not resolved - we don't need the License section, thanks.

**Pull request closed without merging**